util/mkconf.c

```diff
@@ -0,0 +1,142 @@
+#include <string.h>
+#include <stdio.h>
+
+struct cfgopt {
+       const char *name;
+       const char *macro;
+       const char *help;
+     };
+
+enum optval {
+     OPT_UNDEFINED = 0,
+     OPT_ENABLE,
+     OPT_DISABLE
+   };
+
+struct cfgopt options[] =
```

Comment:
A `static struct cfgopt options[] =` IMHO.

Reply:
Good point, missed that.